virttest/qemu_monitor.py

```diff
@@ -2034,16 +2034,17 @@ def live_snapshot(self, device, snapshot_file, **kwargs):
         return self.cmd("blockdev-snapshot-sync", kwargs)
 
     def block_stream(self, device, speed=None, base=None,
-                     cmd="block-stream", correct=True, backing_file=None):
+                     cmd="block-stream", correct=True, **kwargs):
         """
         Start block-stream job;
 
         :param device: device ID
         :param speed: int type, limited speed(B/s)
         :param base: base file
         :param correct: auto correct command, correct by default
-        :param backing_file: backing file
-
+        :param kwargs: optional keyword arguments including but not limited to below
+        :keyword Args:
+                backing_file: backing file
```

Comment:
This docstring will be less likely to be updated in the future, so I suggest hide the details.